File: PHP/OOP/pessoa.php
<?php
    class Pessoa{
        private $nome;
        private $idade;
        private $cidade;

        public function setNome($nome){
            $this->nome = $nome;
        }
        public function getNome(){
            return $this->nome;
        }

        public function setIdade($idade){
            if ($idade > 0) {
                $this->idade = $idade;
                return true;
            }
            else{
                return false;
            }
        }
        public function getIdade(){
            return $this->idade;
        }
    }
    class Livro{
        
    }
?>

File: cs/OOP/pessoa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OOP
{
    public class pessoa
    {
        string nome;
        //endereco end;
        public pessoa(string nome)
        {
            this.nome = nome;
        }
        public string getNome()
        {
            return nome;
        }
        public void alteraNome(string nome)
        {
            this.nome = nome;
        }
    }
}
